application/models/TemplateConfiguration.php

```diff
@@ -982,7 +982,7 @@ public function getOptionPageAttributes()
             foreach ($fileList as $file) {
                 $imageInfo = $this->getImageInfo($basePath . $file['name'], $pathPrefix);
                 if ($imageInfo) {
-                    $aData['imageFileList'][] = array_merge(
+                    $aData['imageFileList'][$imageInfo['filepath']] = array_merge(
```

Comment:
This don't disable usage of 2 filename in different directory ?
For example 

1. Logo.png at global upload directry
2. Logo.png at template directory
3. Logo.png at survey level directory

Reply:
Shouldn't. The key of the array is the path, not the filename.